manifests/docker.pp

```diff
@@ -42,7 +46,8 @@
   }
   ~>
   service { 'docker-xylem':
-    ensure => running,
+    ensure  => running,
+    require => File['/run/docker/plugins'],
```

Comment:
The `~>` above is equivalent to a `subscribe` param and the `->` above that is equivalent to a `require` param. Unfortunately, arrows can't easily indicate parallel dependencies where foo depends on bar and baz, but neither bar nor baz depends on the other.